src/tools/kdb/mv.cpp

```diff
@@ -35,6 +35,12 @@ int MvCommand::execute (Cmdline const & cl)
 	Key sourceKey = cl.createKey (0, false);
 
 	Key destKey = cl.createKey (1, false);
+
+	if (sourceKey.getNamespace () == kdb::ElektraNamespace::CASCADING || sourceKey.getNamespace () == kdb::ElektraNamespace::CASCADING)
```

Comment:
```suggestion
	if (sourceKey.getNamespace () == kdb::ElektraNamespace::CASCADING)
```

Aren't you specifying the same condition twice?

Reply:
Yes, missed it... `= cl.createKey (1, false);`already failed before the check, that's why the test did not fail, I removed those so the error message is more specific.